aws/resource_aws_vpc_endpoint_service.go

```diff
@@ -214,17 +242,39 @@ func resourceAwsVpcEndpointServiceRead(d *schema.ResourceData, meta interface{})
 		return fmt.Errorf("error setting allowed_principals: %s", err)
 	}
 
+	err = d.Set("private_dns_name_config", flattenPrivateDnsNameConfiguration(svcCfg.PrivateDnsNameConfiguration))
+	if err != nil {
+		return fmt.Errorf("error setting private_dns_name_config: %s", err)
```

Comment:
Nit. Prefer `%w` over `%s` when returning a wrapped error.

```suggestion
		return fmt.Errorf("error setting private_dns_name_config: %w", err)
```